Tools/langtool/unused-euristic.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+langfile='../../assets/lang/en_US.ini'
+folders=('../../UI/' '../../Core/' '../../Common/' '../../Windows/' '../../assets/shaders' '../../assets/themes')
```

Comment:
It seems like this is missing GPU?  Intentional?

-[Unknown]

Reply:
Nope, my bad :/